entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -144,3 +144,7 @@ Sometimes you may want to reference types from another DbContext. This can lead
 ### Next steps
 
 The above was only a brief introduction to migrations. Please consult the other documentation pages to learn more about [managing migrations](xref:core/managing-schemas/migrations/managing), [applying them](xref:core/managing-schemas/migrations/applying), and other aspects. The [.NET Core CLI tool reference](xref:core/miscellaneous/cli/index) also contains useful information on the different commands
+
+## Links
```

Comment:
Can we consistently title this as "Additional resources"?

Reply:
Sure, will change.